dist/qs.js

```diff
@@ -80,36 +80,38 @@ var parseValues = function parseQueryStringValues(str, options) {
     return obj;
 };
 
-var parseObject = function parseObjectRecursive(chain, val, options) {
-    if (!chain.length) {
-        return val;
-    }
+var parseObject = function (chain, val, options) {
```

Comment:
regardless, all of these changes where you removed function names would have to be reverted.

Reply:
I don't really understand where this change comes from... My changes in lib are restricted to the object of this PR...

Reply:
You might be rebased on a much older version of master?

Reply:
Oh, also, the dist file should never be updated except during a release :-) you can just revert this whole file.